.lintstagedrc.json

```diff
@@ -1,5 +1,6 @@
 {
   "*.{js,ts}": ["eslint --fix", "prettier --write"],
   "*.vue": ["eslint --fix", "stylelint --fix", "prettier --write"],
-  "*.json": ["prettier --write"]
+  "*.json": ["prettier --write"],
+  "*.md": ["textlint"]
```

Comment:
これは意図的ではないです。`README.md` はべた書き HTML などを含んでいて textlint で処理するべきではないものとして判断したので弾きました。

```suggestion
  "src/content/*.md": ["textlint"]
```

にしたら動くんでしょうか？`*` でも再帰的にファイル精査してくれるんですかね？globなら `src/content/**/*.md` にしなきゃならなそうですが…。

Reply:
`README.md` を弾く理由がガバい気もするので、`README.md` も textlint の対象とするべきという意見であれば受け入れます。

Reply:
https://zenn.dev/kira924age/articles/98bc03d898c706#lint-staged-%E3%81%AE%E8%A8%AD%E5%AE%9A
`**` じゃないとダメそうかも…

Reply:
ちょっとあとで本家のマニュアル見ます

Reply:
https://github.com/okonet/lint-staged#filtering-files

> - If the glob pattern contains no slashes (`/`), micromatch's `matchBase` option will enabled, so globs match a file's basename regardless of directory:
>   - **`"*.js"`** will match all JS files, like `/test.js` and `/foo/bar/test.js`
>   - **`"!(*test).js"`**. will match all JS files, except those ending in `test.js`, so `foo.js` but not `foo.test.js`
> - If the glob pattern does contain a slash (`/`), it will match for paths as well:
>   - **`"./*.js"`** will match all JS files in the git repo root, so `/test.js` but not `/foo/bar/test.js`
>   - **`"foo/**/*.js"`** will match all JS files inside the `/foo` directory, so `/foo/bar/test.js` but not `/test.js`

```suggestion
  "src/content/**/*.md": ["textlint"]
```

かな？